src/AaveV3Update_20231002weth/AaveV3Update_20231002weth.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IEngine,EngineFlags,Rates} from 'aave-helpers/v3-config-engine/AaveV3PayloadBase.sol';
+import {
+  AaveV3PayloadOptimism,
+  AaveV3OptimismAssets
+} from 'aave-helpers/v3-config-engine/AaveV3PayloadOptimism.sol';
+import {
+  AaveV3PayloadArbitrum,
+  AaveV3ArbitrumAssets
+} from 'aave-helpers/v3-config-engine/AaveV3PayloadArbitrum.sol';
+
+contract AaveV3OptimismUpdate20231002wethPayload is AaveV3PayloadOptimism {
```

Comment:
we recommend not including mutlichain contracts in a single file. There is no dead code elimination, so foundry etherscan verification will take the whole file, making it hard to review the onchain deployment on etherscan (as it will contain everything, not only optimism in this case)